extensions/ql-vscode/src/interface-utils.ts

```diff
@@ -128,8 +129,13 @@ export function getHtmlForWebview(
   // Use a nonce in the content security policy to uniquely identify the above resources.
   const nonce = getNonce();
 
-  const stylesheetsHtmlLines = stylesheetWebviewUris.map(stylesheetWebviewUri =>
-    `<link nonce="${nonce}" rel="stylesheet" href="${stylesheetWebviewUri}">`);
+  const stylesheetsHtmlLines = allowInlineStyles
+    ? stylesheetWebviewUris.map(uri => createStylesLinkWithoutNonce(uri))
+    : stylesheetWebviewUris.map(uri => createStylesLinkWithNonce(nonce, uri));
+
+  const styleSrc = allowInlineStyles
+    ? 'https://*.vscode-webview.net/ vscode-file: \'unsafe-inline\''
```

Comment:
Does this mean that we will be downloading css from `vscode-webview.net`? If so, then we need to make sure that local queries still work even if there's no internet connection. I think that's fine since only the remote queries viewer is using the inline styles.

Reply:
I've tested running normal queries without internet connection and the view looks + styles are loading.